### PR TITLE
Rfc readme pr

### PR DESCRIPTION
**Pull request by jakubcech**
PR for changing the RFC template motivation description as discussed in the following RFC:
https://github.com/iotaledger/bee-rfcs/pull/15

### CODE REVIEW

**Review by wusyong — APPROVED** [review]
LGTM

**Review by Alex6323 — APPROVED** [review]
I very much like those changes to the RFC process. With those the RFC author has to come up with convincing use-cases for his proposal, which may make him reevaluate his thought process, and make it easier for reviewers to decide on its usability. Additionally, if a proposal covers too many or too few use-cases, this might be a good indicator to split it up into multiple, or merge it with some other RFC.

File: 0000-template.md [hunk 1]
```diff
@@ -11,6 +11,21 @@ One paragraph explanation of the feature.
...
+
+When _____ , I want to _____ , so I can _____ .
+
+Example 1:
+When I query a node for a list of transactions, I want to be able to sort them by date, so I can work with the most relevant ones.
+
+Example 2:
+When I configure a node, I want to be able to control how much transaction history the node stores, so I can make sure I only store the data I need without incurring additional operational costs.
+
+```
```
Comment (thibault-martinez) [inline 0000-template.md]: Formatting error ?

**Review by thibault-martinez — CHANGES_REQUESTED** [review]

**Review by tsvisabo — APPROVED** [review]

**Review by bingyanglin — APPROVED** [review]

File: 0000-template.md (hunk 1 above)
Comment (jakubcech) [reply 0000-template.md]: Good catch, leftovers from the past. :) Fixed.

**Review by thibault-martinez — APPROVED** [review]

**Comment by SuperFluffy** [comment]
I have pushed some extra formatting. In particular, I replaced the `___` underscores by `FULLWIDTH LOW LINE (U+FF3F)`, i.e. the character `＿`, because they were messing with the formatting of my markdown viewer (vim), and I reformatted the documented to introduce linebreaks.

Thank you very much for the PR, @jakubcech.

**Pull request merged**